python/cugraph-service/client/cugraph_service_client/client.py

```diff
@@ -32,6 +32,32 @@
 from cugraph_service_client.cugraph_service_thrift import create_client
 
 
+class RunAsyncioThread(threading.Thread):
+    def __init__(self, func, args, kwargs):
+        self.func = func
+        self.args = args
+        self.kwargs = kwargs
+        self.result = None
+        super().__init__()
+
+    def run(self):
+        self.result = asyncio.run(self.func(*self.args, **self.kwargs))
+
+
+def run_async(func, *args, **kwargs):
```

Comment:
This is nice. Can you also add a simple docstring to this function though (and possibly one for the `RunAsyncioThread` class too for completeness)?